queries.sql

```diff
@@ -9,3 +9,47 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT  IN ('Gabumon');
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+
+
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+
```

Comment:
- [ ] Good job on this query but please you should verify that the change persists after commit.